assets/ZZZ/components/ui/entity-list/use-reorder-entities.js

```diff
@@ -1,113 +0,0 @@
-/**
```

Comment:
@tn3rb , not sure if we will need this file since we have this: https://github.com/eventespresso/event-espresso-core/blob/EDTR/master/assets/src/domain/shared/entities/datetimes/predicates/sorters/index.ts

Reply:
missed this comment during my first review...
ya if we can use shared predicates, then that would be preferred

Reply:
cool, it's has been deleted, wanted to double-check with you